com.interversehq.qview.yml

```diff
@@ -0,0 +1,18 @@
+app-id: com.interversehq.qView
```

Comment:
The file name needs to be the same as the app ID.

Reply:
Fixed.